src/test/java/seedu/hirehub/testutil/TypicalPersons.java

```diff
@@ -55,7 +55,8 @@ public class TypicalPersons {
 
     public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
 
-    private TypicalPersons() {} // prevents instantiation
+    private TypicalPersons() {
+    } // prevents instantiation
```

Comment:
I would prefer that the braces are on the same line to standardize with methods in other classes.